example/model-server/mms.md

```diff
@@ -0,0 +1,115 @@
+# Model Server for Apache MXNet (incubating)
```

Comment:
Delete. This is MMS code from AWSLabs, and its documentation shouldn't reside in MXNet.

Reply:
I tend to agree with @szha - isn't it enough to link to MMS repo readme (or to modelserver.io) from ecosystem.md?
Duplicating docs means more effort to maintain, keep up to date, etc.

Reply:
I actually summarized quite a bit of info in this one page. Maybe I'll just put it in a blog. No content should be wasted.

Reply:
Also, it's not like I was going to just do this for MMS. I hope to add landing pages with helpful usage info for the NLP and CV toolkits and others as I get time to work with them. It's not about duplication. As you can see in this MMS doc, I cross link to the relevant docs. Cross-linking is important for SEO. But... whatever. I'll put it elsewhere if you are dead-set against having it.